src/main/java/reposense/system/CommandRunner.java

```diff
@@ -54,16 +54,12 @@ public static void checkoutToDate(String root, String branchName, Date untilDate
         }
 
         Path rootPath = Paths.get(root);
-        String checkoutCommand;
+
         String substituteCommand = "git rev-list -1 --before="
                 + GIT_LOG_UNTIL_DATE_FORMAT.format(untilDate) + " " + branchName;
+        String hash = runCommand(rootPath, substituteCommand);
 
-        if (isWindows) {
-            checkoutCommand = "for /f %g in ('" + substituteCommand + "') do git checkout %g";
-        } else {
-            checkoutCommand = "git checkout `" + substituteCommand + "`";
-        }
-
+        String checkoutCommand = "git checkout " + hash;
```

Comment:
hmm, true.

instead of ` so we will just do full repo analyze instead (since we won't checkout to any commit here)`

Shall we do a `git checkout --orphan NEWBRANCH` to perform an `empty analysis` instead?
Because if we do full repo analysis, the result will be misaligned with the commits.

Reply:
Hmmm, best if we could just completely skip this repo, but yea doing an `empty analysis` should work as well.